code/WorkInProgress/ItemSpecials.dm

```diff
@@ -597,10 +602,22 @@
 							A.attackby(master, user, params, 1)
 							attacked += A
 							hit = 1
+					if(ignition)
+						T.hotspot_expose(3000,1)
+						for(var/A in T)
+							if(ismob(A))
+								var/mob/M = A
+								M.changeStatus("burning", 8 SECONDS)
+							else if(iscritter(A))
+								var/obj/critter/crit = A
+								crit.blob_act(8) //REMOVE WHEN WE ADD BURNING OBJCRITTERS
 
 				afterUse(user)
 				if (!hit)
-					playsound(get_turf(master), 'sound/effects/swoosh.ogg', 50, 0)
+					if (!ignition)
+						playsound(get_turf(master), 'sound/effects/swoosh.ogg', 50, 0)
+					else
+						playsound(get_turf(master), 'sound/effects/flame.ogg', 50, 0)
```

Comment:
Change `playsound` use the sound cache instead. I think every sound you use here should already be available in the cache.
```suggestion
						playsound(get_turf(master), "sound/effects/swoosh.ogg", 50, 0)
					else
						playsound(get_turf(master), "sound/effects/flame.ogg", 50, 0)
```

Reply:
So just replace the `'`s with `"`s?

Reply:
So long as the sound file is in the queue yes. Otherwise you need to add them which is usually done by running `buildSoundList.ps1`

Reply:
Done.